docker/scripts/deprovision.sh

```diff
@@ -271,12 +271,21 @@ baremetal_2a2 | baremetal_2a4 | baremetal_hua)
 *)
 	set_autofail_stage "running packet-hardware inventory"
 	packet-hardware inventory --verbose --tinkerbell "${tinkerbell}/hardware-components"
+	if [[ $arch == "x86_64" ]]; then
```

Comment:
Can you add a comment as to why the bios is only inventoried for x86-64? It isn't obvious from the context why, so this behavior might get refactored out later.

Reply:
Done.